resources/js/Pages/Auth/Verify-Email.vue

```diff
@@ -1,15 +1,16 @@
 <script setup lang="ts">
```

Comment:
Can we change filename to `VerifyEmail`?

Reply:
> Can we change filename to `VerifyEmail`?
done

Reply:
done